src/FiniteElementDiscretization.cxx

```diff
@@ -229,4 +229,22 @@ namespace mfem_mgis {
     return fed.getFiniteElementSpace<false>().GetTrueVSize();
   }  // end of getTrueVSize
 
+  std::shared_ptr<Mesh<false>> loadMeshSequential(const std::string& mesh_name,
+                                                  int generate_edges,
+                                                  int refine,
+                                                  bool fix_orientation) {
+#ifdef MFEM_USE_MED
+    if (std::string("med").compare(getFileExt(mesh_name)) == 0) {
```

Comment:
`getFileExt(mesh_name)=="med"` is much clearer and efficient, isnt'it ?

Reply:
Done